extensions/http-curl/tests/HTTPHandlers.h

```diff
@@ -422,11 +423,15 @@ class HeartbeatHandler : public ServerAwareHandler {
 
   void verify(struct mg_connection *conn) {
     auto post_data = readPayload(conn);
+    if (!isServerRunning()) {
+      return;
+    }
     if (!IsNullOrEmpty(post_data)) {
       rapidjson::Document root;
-      rapidjson::ParseResult ok = root.Parse(post_data.data(), post_data.size());
-      assert(ok);
-      (void)ok;  // unused in release builds
+      rapidjson::ParseResult result = root.Parse(post_data.data(), post_data.size());
+      if (!result) {
+        throw std::runtime_error("JSON parse error: " + std::string(rapidjson::GetParseError_En(result.Code())) + "\n JSON data: " + post_data);
```

Comment:
`fmt::format` or `StringUtils::join_pack` would be nicer IMO. `fmt::format` allows nice format string. If you don't like that, `StringUtils::join_pack` can avoid intermediate allocation and deallocation of temporaries.

Reply:
Updated in a199f6907d09938d30af9fe21b65ffc0e5116cb2